### PR TITLE
libbpf-tools/sigsnoop: Allow to trace a signal list

### PR DESCRIPTION
**Pull request by ZhangYet**
tools/killsnoop supports a signal list(47d7263ec4bb8163c0c5fac70edaca9f1b2307f6), 
it's reasonable to support the same feature in libbpf-tools/sigsnoop.

### CODE REVIEW

**Comment by ZhangYet** [comment]
@brendangregg @devidasjadhav @yonghong-song

File: libbpf-tools/sigsnoop.bpf.c [hunk 1]
```diff
@@ -72,7 +84,6 @@ int kill_entry(struct syscall_trace_enter *ctx)
 {
 	pid_t tpid = (pid_t)ctx->args[0];
 	int sig = (int)ctx->args[1];
-
```
Comment (yonghong-song) [inline libbpf-tools/sigsnoop.bpf.c:75]: Please keep this empty line here. We would like an empty line between declaration and other codes.

File: libbpf-tools/sigsnoop.c [hunk 2]
```diff
@@ -78,24 +100,30 @@ const char argp_program_doc[] =
...
 	{ "failed", 'x', NULL, 0, "Trace failed signals only.", 0 },
 	{ "kill", 'k', NULL, 0, "Trace signals issued by kill syscall only.", 0 },
 	{ "pid", 'p', "PID", 0, "Process ID to trace", 0 },
-	{ "signal", 's', "SIGNAL", 0, "Signal to trace.", 0 },
+	{ "signal", 's', "SIGNAL", 0, "Signals to trace.", 0 },
 	{ "name", 'n', NULL, 0, "Output signal name instead of signal number.", 0 },
 	{ "verbose", 'v', NULL, 0, "Verbose debug output", 0 },
 	{ NULL, 'h', NULL, OPTION_HIDDEN, "Show the full help", 0 },
 	{},
+>>>>>>> 7c8106a2 (libbpf-tools/sigsnoop: Allow to trace a signal list)
```
Comment (yonghong-song) [inline libbpf-tools/sigsnoop.c:114]: Here and below as well.

**Review by yonghong-song — COMMENTED** [review]
You need to resolve conflicts before submitting the pull request.

File: libbpf-tools/sigsnoop.c (hunk 2 above)
Comment (ZhangYet) [reply libbpf-tools/sigsnoop.c]: Sorry, my fault. Fixed.

File: libbpf-tools/sigsnoop.bpf.c (hunk 1 above)
Comment (ZhangYet) [reply libbpf-tools/sigsnoop.bpf.c]: Fixed.

**Comment by ZhangYet** [comment]
> You need to resolve conflicts before submitting the pull request.

I have fixed them. Thanks.

**Pull request merged**